PypeS/pype.py

```diff
@@ -236,20 +238,18 @@ def Execute(self):
         for scriptNameAndArguments in self.ScriptList:
             self.PrintLog('')
             scriptName = scriptNameAndArguments[0]
-            moduleName = scriptName
             scriptArguments = scriptNameAndArguments[1]
             imported = True
             try:
-                exec('import '+ moduleName)
-            except ImportError, e:
-                self.PrintError(e.message)
+                module = importlib.import_module('vmtk.'+scriptName)
+                moduleName = module.__name__
+                scriptObjectClasses = [x for x in dir(module) if isclass(getattr(module, x))]
+                scriptObjectClassName = scriptObjectClasses[0]
+            except ImportError as e:
+                self.PrintError(str(e))
                 break
-            scriptObjectClassName = ''
-            exec ('scriptObjectClassName =  '+moduleName+'.'+moduleName)
-            moduleScriptObjectClassName = moduleName+'.'+scriptObjectClassName
-            self.PrintLog('Creating ' + scriptObjectClassName + ' instance.')
-            scriptObject = 0
-            exec ('scriptObject = '+moduleScriptObjectClassName+'()')
+            scriptObject = getattr(module, scriptObjectClassName)
+            scriptObject = scriptObject()
```

Comment:
I'm all for this.

The only potential issue is getting the first class that comes across. There are script modules that contain multiple classes (e.g. vmtkcenterlines.py) that would violate the assumption.

Instead, we should get the first class that is a descendant of `pypes.pypeScript`, making the reasonable assumption that there would be just one per module.

The would remove the need for having a line like
`vmtkcenterlines = 'vmtkCenterlines'`
in every script, which was a total hack. The code assumes that there is one variable in the module named as the module (i.e. `vmtkcenterlines`), whose value is the name of the class. The `exec ('scriptObjectClassName =  '+moduleName+'.'+moduleName)` therefore assigns the name of the class to the `scriptObjectClassName`.
With your change, we should also remove all such variables from scripts.